README.md

```diff
@@ -107,11 +107,18 @@ pip install -e ".[dev]"
 ```
 and follow the more detailed instructions in [CONTRIBUTING](https://github.com/facebookresearch/theseus/blob/main/CONTRIBUTING.md).
 
+Compilation from source supports is affected by the following enviroment variables:
+* `BASPACHO_ROOT_DIR`: root dir of [BaSpaCho](https://github.com/facebookresearch/baspacho) sparse solver, which must be already
+  built with binaries in the subdirectory `build`.
+* `THESEUS_ENABLE_CUDA` (1 or 0): if set, forces enabling or disabling Cuda support regardless of system's `torch.cuda.is_available()`.
+
 ### Running unit tests (requires `dev` installation)
 ```bash
 python -m pytest theseus
 ```
-By default, unit tests include tests for our CUDA extensions. You can add the option `-m "not cudaext"` to skip them when installing without CUDA support.
+By default, unit tests include tests for our CUDA extensions, and for BaSpaCho sparse solver. You can add the option `-m "not cudaext"`
+to skip them when installing without CUDA support, or `-m "not baspacho"` installing without BaSpaCho (or
+`-m "(not cudaext) and (not baspacho)"` to disable both).
```

Comment:
Should we have the cpu baspacho alwasy included? Then we can subsume cuda baspacho under all cuda stuff with cudaext and avoid having multiple flags.

Reply:
it is included in tests by default, indeed, and that's why I added a marker to run tests not depending on it.

Reply:
I think @mhmukadam is making a different point, which is to have only the one flag for CUDA (which would include LUCuda, but also CUDA+Baspacho), and then **always** install CPU-Baspacho, so that we don't need the other flag at all. Is this correct?

I'm not sure about this change because BASPACHO support requires a manual installation of a dependency (the core BASPACHO), so it seems a bit too much user work for it to be considered the default behavior. 

I would actually propose to remove the "baspacho" marker, and just change unit tests to `try` catch the Baspacho import, and only emit a warning that Baspacho is not installed, instead of crashing.

Reply:
Yes, one flag for CUDA is what I meant.

Reply:
Not sure if I'm understanding. If BaSpaCho is installed, there is just one flag indeed `-m cudaext` runs Cuda tests (including the BaSpaCho ones), and `-m "not cudaext` runs the CPU tests (including BaSpaCho ones).
Now, if you decide to install without BaSpaCho (which is currently still possible) you might want to run tests skipping the BaSpaCho ones, and `-m "not baspacho"` allows you to do exactly that. If BaSpaCho is installed, the only relevant marker is `cudaext`.

Reply:
done, now tests are skipped with a "SKIPPED" message if `hasattr(theseus.extlib, "baspacho_solver")` fails.